openedx/core/djangoapps/cors_csrf/tests/test_middleware.py

```diff
@@ -269,7 +269,9 @@ def _assert_cookie_sent(self, response, is_set):
                 domain=self.COOKIE_DOMAIN
             )
             self.assertIn(expected, cookie_header)
-            self.assertIn('Max-Age=31449600; Path=/; secure', cookie_header)
+            self.assertIn('Max-Age=31449600;', cookie_header)
+            self.assertIn('Path=/;', cookie_header)
+            self.assertIn('secure', cookie_header.lower())
```

Comment:
@moeez96 Please replace these three lines with a single line referenced from upstream 
https://github.com/edx/edx-platform/blob/8c43855cfe6660a484e80c0b6540200bd263f52a/openedx/core/djangoapps/cors_csrf/tests/test_middleware.py#L275

Reply:
We have seen that tests fail in that case https://app.circleci.com/pipelines/github/edx-solutions/edx-platform/527/workflows/d347c3c8-8e78-47cc-a639-99fe59cf0cac/jobs/5483/parallel-runs/1?filterBy=FAILED

Reply:
@moeez96 the link you shared has the different code at that line. Please compare that line with the one I shared above and change that line accordingly.